packages/store-ui/src/atoms/Button/Button.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Story, Preview, ArgsTable } from "@storybook/addon-docs/blocks";
+import Button from "./Button";
+
+<Meta title="Atoms/Button" />
+
+# Preview
+
+<Preview>
+  <Story name="Button" args={{
+    children: "Button"
```

Comment:
why not calling it "label" and then (args) => <Button {...args}>{args.label}</Button>

Reply:
Because we don't have a "label" prop, this data is literally the children. We'll try to follow exactly the prop of ArgsTable and what it means.